Formula/tesseract-lang.rb

```diff
@@ -0,0 +1,25 @@
+class TesseractLang < Formula
+  desc "Enables extra languages support for Tesseract"
+  homepage "https://github.com/tesseract-ocr/tessdata_fast/"
+  url "https://github.com/tesseract-ocr/tessdata_fast/archive/4.0.0.tar.gz"
+  sha256 "f1b71e97f27bafffb6a730ee66fd9dc021afc38f318fdc80a464a84a519227fe"
+
+  depends_on "tesseract"
+
+  def install
+    rm "eng.traineddata"
+    rm "osd.traineddata"
+    (share/"tessdata").install Dir.glob("./*")
+  end
+
+  test do
+    resource "tests" do
+      url "https://github.com/tesseract-ocr/test.git"
+    end
+
+    resource("tests").stage do
+      system "#{Formula["tesseract"].bin}/tesseract", "./testing/eurotext.tif", "./output", "-l", "eng+deu"
+      assert_equal "über den faulen Hund. Le renard brun\n", shell_output("sed -n 6p ./output.txt")
```

Comment:
I would suggest to put also for the French part "something" that uses more that "English" letters, e.g. `Académie française`.

Reply:
Do you mean using another image from the test repository? Any suggestions?

Reply:
Yes, something that uses more specific French letters than `Le renard brun`. (In the German part you have at least one Umlaut: `über`.)

Reply:
I could add another line of the same `eurotext.tif` image, but that Umlaut is exactly the point of this test. The fact that `tesseract` correctly used the `deu` language to recognize the ū proves that the language packs were installed correctly. I do not think that is advantageous to test also for French (or for one of the other 160 languages installed), but I can add more languages to this block if Homebrew maintainers think that is necessary. After all, none of these tests existed in the current version of the formula.

Reply:
The test looks fine as it is.
Please use `assert_match` directly on the content of output, rather than call `sed`.

Reply:
I understand, thank you.

Reply:
@fxcoudert I am not sure I understood your suggestion. I removed `sed` in favor of `cat` and used `assert_match`, is that ok?